src/macros.rs

```diff
@@ -688,21 +688,168 @@ mod syntax_test {
 
 #[cfg(test)]
 mod functional_test {
+    use {client, server};
     use futures::{Future, failed};
+    use std::io;
+    use std::net::SocketAddr;
     use util::FirstSocketAddr;
     extern crate env_logger;
 
+    macro_rules! t {
```

Comment:
This is basically unwrap on steroids, right? Is there a mnemonic to remember that?